core/common/src/main/java/alluxio/exception/AlluxioException.java

```diff
@@ -20,7 +20,7 @@
  * the RPC framework and convert back without losing any necessary information.
  */
 @ThreadSafe
-public abstract class AlluxioException extends Exception {
+public class AlluxioException extends Exception {
```

Comment:
What do you think of `UnexpectedAlluxioException`? I still think `AlluxioRuntimeException` sounds too much like it extends `RuntimException`